source/install/desktop.rst

```diff
@@ -88,16 +88,37 @@ Unofficial, community-driven ``.deb`` packages are available.
 1. Download the latest version of the Mattermost Desktop App:
 
  - 64-bit systems
-   `mattermost-desktop-4.6.2-linux-amd64.deb <https://releases.mattermost.com/desktop/4.6.2/mattermost-desktop-4.6.2-linux-amd64.deb>`__
+   `mattermost-desktop-4.7.0-linux-amd64.deb <https://releases.mattermost.com/desktop/4.7.0/mattermost-desktop-4.7.0-linux-amd64.deb>`__
  - 32-bit systems
-   `mattermost-desktop-4.6.2-linux-i386.deb <https://releases.mattermost.com/desktop/4.6.2/mattermost-desktop-4.6.2-linux-i386.deb>`__
+   `mattermost-desktop-4.7.0-linux-i386.deb <https://releases.mattermost.com/desktop/4.7.0/mattermost-desktop-4.7.0-linux-i386.deb>`__
 
 2. At the command line, execute one of the following commands depending on the package that you downloaded:
 
  - 64-bit systems
-    ``sudo dpkg -i mattermost-desktop-4.6.2-linux-amd64.deb``
+    ``sudo dpkg -i mattermost-desktop-4.7.0-linux-amd64.deb``
  - 32-bit systems
-    ``sudo dpkg -i mattermost-desktop-4.6.2-linux-i386.deb``
+    ``sudo dpkg -i mattermost-desktop-4.7.0-linux-i386.deb``
+
+3. To run Mattermost, open **Dash** (located at the top left corner), enter **mattermost**, then click the Mattermost icon.
+
+Linux rpm files (beta)
+~~~~~~~~~~~~~~~~~~~~~~
+
+Unofficial, community-driven ``.rpm`` packages are available.
+
+1. Download the latest version of the Mattermost Desktop App:
+
+ - 64-bit systems
+   `mattermost-desktop-4.7.0-linux-i686.rpm <https://releases.mattermost.com/desktop/4.7.0/mattermost-desktop-4.7.0-linux-i686.rpm>`__
+ - 32-bit systems
+   `mattermost-desktop-4.7.0-linux-x86_64.rpm <https://releases.mattermost.com/desktop/4.7.0/mattermost-desktop-4.7.0-linux-x86_64.rpm>`__
+
+2. At the command line, execute one of the following commands depending on the package that you downloaded:
+
+ - 64-bit systems
+    ``sudo dpkg -i mattermost-desktop-4.7.0-linux-i686.rpm``
+ - 32-bit systems
+    ``sudo dpkg -i mattermost-desktop-4.7.0-linux-x86_64.rpm``
```

Comment:
I added rpm here, but need help with checking for accuracy.

Reply:
```suggestion
Linux rpm files (beta)
~~~~~~~~~~~~~~~~~~~~~~

Unofficial, community-driven ``.rpm`` packages are available.

1. Download the latest version of the Mattermost Desktop App:

 - 32-bit systems
   `mattermost-desktop-4.7.0-linux-i686.rpm <https://releases.mattermost.com/desktop/4.7.0/mattermost-desktop-4.7.0-linux-i686.rpm>`__
 - 64-bit systems
   `mattermost-desktop-4.7.0-linux-x86_64.rpm <https://releases.mattermost.com/desktop/4.7.0/mattermost-desktop-4.7.0-linux-x86_64.rpm>`__

2. At the command line, execute one of the following commands depending on the package that you downloaded:

 - 32-bit systems
    ``sudo rpm -i mattermost-desktop-4.7.0-linux-i686.rpm``
 - 64-bit systems
    ``sudo rpm -i mattermost-desktop-4.7.0-linux-x86_64.rpm``
```

I might be wrong but I think that's the right command. Also fixed the architecture